src/main/kotlin/org/wycliffeassociates/otter/jvm/usecases/CreateProjectUseCase.kt

```diff
@@ -0,0 +1,57 @@
+package org.wycliffeassociates.otter.jvm.usecases
+
+import io.reactivex.Single
+import io.reactivex.rxjavafx.schedulers.JavaFxScheduler
+import org.wycliffeassociates.otter.common.data.model.Collection
+import org.wycliffeassociates.otter.common.data.model.Language
+import org.wycliffeassociates.otter.common.data.model.Resource
+import org.wycliffeassociates.otter.common.data.model.SourceCollection
+import org.wycliffeassociates.otter.jvm.persistence.repositories.CollectionRepository
+import org.wycliffeassociates.otter.jvm.persistence.repositories.LanguageRepository
+import org.wycliffeassociates.otter.jvm.persistence.repositories.ProjectRepository
+import org.wycliffeassociates.otter.jvm.persistence.repositories.SourceRepository
+
+
+class CreateProjectUseCase(val languageRepo: LanguageRepository,
```

Comment:
It might be best to move this to the common repo. All the repo interfaces should be there, so this could go in the domain package in common.